cli/src/test/java/com/liferay/blade/cli/UtilTest.java

```diff
@@ -154,6 +155,55 @@ public void testIsWorkspace3() throws Exception {
 		Assert.assertTrue(WorkspaceUtil.isWorkspace(workspace));
 	}
 
+	@Test
+	public void testMigrateBladeSettings() throws Exception {
```

Comment:
I see in this method you are testing the 'non-migrate' test case (in the first half of the method) and then testing the 'migrate' test case.  Can you split these two up into two separate methods?